css/bem-css.md

```diff
@@ -38,58 +40,58 @@ We do this:
 
 We _don't_ do this:
 ```css
-.feature
-.content
-.btn
+.c-feature
+.c-content
+.c-btn
 ```
 
 We do this:
 ```css
-.alert-box
-.button
+.c-alert-box
+.c-button
 ```
 
 **Element**: styles that only apply to children of a block. Elements can also be blocks themselves. Class name is a concatenation of the block name, two underscores and the element name. Examples:
```

Comment:
Agreed that is confusing. I'll just take it out as I think it is explained how to tackle this further down at line 76